src/Detector/Go/GoPlatformDetectorResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+
+namespace Microsoft.Oryx.Detector.Go
+{
+    public class GoPlatformDetectorResult : PlatformDetectorResult
+    {
+        /// <summary>
+        /// Gets or sets the value indicating if a 'go.mod' file exists in the repo.
+        /// </summary>
+        public bool goDotModExists { get; set; }
```

Comment:
Rename to `GoDotModExists` since this is a public property?

Reply:
👍